doc/02-Twig-Integration.markdown

```diff
@@ -181,6 +181,8 @@ menu object in the other functions:
 {{ knp_menu_render(['main', 'Comments', 'My comments'], {'depth': 2}) }}
 
 {% set item = knp_menu_get('sidebar', ['First section']) %}
+
+{% set breacrumbs_array = knp_menu_get_breadcrumbs_array('main') %}
```

Comment:
Yes, this simply returns an array and #161 is about rendering the array with a renderer class instead of doing it in a template.